File: pgm/ABC_Dart/ABC_415_A.dart
// import 'dart:collection';
import 'dart:io';
// import 'dart:math';

void main() {
  stdin.readLineSync()!;
  final List<int> A = stdin.readLineSync()!.split(' ').map(int.parse).toList();
  final int X = int.parse(stdin.readLineSync()!);
  final String Yes = 'Yes';
  final String No = 'No';
  if (A.contains(X)) {
    print(Yes);
  } else {
    print(No);
  }
  exit(0);
}

File: pgm/ABC_C#/ABC_415_A.cs
using System;
// using System.Collections.Generic;
// using System.Globalization;
// using System.IO;
using System.Linq;
// using System.Text;
// using System.Text.RegularExpressions;

class Program
{
    static void Main()
    {
        int N = int.Parse(Console.ReadLine());
        int[] A =
            Array.ConvertAll(Console.ReadLine().Trim().Split(' '), int.Parse);
        int X = int.Parse(Console.ReadLine());
        const String Yes = "Yes";
        const String No = "No";
        Console.WriteLine(A.Contains(X) ? Yes : No);
        Environment.Exit(0);
    }
}